Formula/couchdb.rb

```diff
@@ -5,6 +5,7 @@ class Couchdb < Formula
   mirror "https://archive.apache.org/dist/couchdb/source/3.2.2/apache-couchdb-3.2.2.tar.gz"
   sha256 "69c9fd6f80133557f68a02e92dda72a4fd646d646f429f45bb8329a30f82f20e"
   license "Apache-2.0"
+  revision 1
```

Comment:
If preferred, we could do the modification in a separate PR and avoid rebuilding bottles as `erlang` is a build-only dependency

Reply:
Cherry-picked into #106767. Can drop commits once that PR is in.